lisa/commands.py

```diff
@@ -3,26 +3,28 @@
 from argparse import Namespace
 from typing import Dict, List, Optional, cast
 
-from lisa.common.logger import log
-from lisa.core.environment_factory import environment_factory
+from lisa.core.environmentFactory import EnvironmentFactory
 from lisa.core.package import import_module
-from lisa.core.platform_factory import platform_factory
+from lisa.core.platformFactory import PlatformFactory
 from lisa.core.runtimeObject import RuntimeObject
-from lisa.core.test_factory import test_factory
+from lisa.core.testFactory import TestFactory
 from lisa.parameter_parser.parser import parse
 from lisa.test_runner.lisarunner import LISARunner
 from lisa.util import constants
+from lisa.util.logger import log
 
 
-def _load_extends(extends_config: Dict[str, object]):
+def _load_extends(base_path: str, extends_config: Optional[Dict[str, object]]) -> None:
     if extends_config is not None:
         paths = cast(List[str], extends_config.get("paths"))
         if paths is not None:
```

Comment:
This is superfluous because if `paths is None` then `for path in paths` is a no-op.